examples/jsm/lines/LineSegmentsGeometry.js

```diff
@@ -9,29 +9,30 @@ import {
 	WireframeGeometry
 } from "../../../build/three.module.js";
 
-var LineSegmentsGeometry = function () {
+class LineSegmentsGeometry extends InstancedBufferGeometry {
 
-	InstancedBufferGeometry.call( this );
+	constructor() {
 
-	this.type = 'LineSegmentsGeometry';
+		super();
 
-	var positions = [ - 1, 2, 0, 1, 2, 0, - 1, 1, 0, 1, 1, 0, - 1, 0, 0, 1, 0, 0, - 1, - 1, 0, 1, - 1, 0 ];
-	var uvs = [ - 1, 2, 1, 2, - 1, 1, 1, 1, - 1, - 1, 1, - 1, - 1, - 2, 1, - 2 ];
-	var index = [ 0, 2, 1, 2, 3, 1, 2, 4, 3, 4, 5, 3, 4, 6, 5, 6, 7, 5 ];
+		this.type = 'LineSegmentsGeometry';
 
-	this.setIndex( index );
-	this.setAttribute( 'position', new Float32BufferAttribute( positions, 3 ) );
-	this.setAttribute( 'uv', new Float32BufferAttribute( uvs, 2 ) );
+		var positions = [ - 1, 2, 0, 1, 2, 0, - 1, 1, 0, 1, 1, 0, - 1, 0, 0, 1, 0, 0, - 1, - 1, 0, 1, - 1, 0 ];
+		var uvs = [ - 1, 2, 1, 2, - 1, 1, 1, 1, - 1, - 1, 1, - 1, - 1, - 2, 1, - 2 ];
+		var index = [ 0, 2, 1, 2, 3, 1, 2, 4, 3, 4, 5, 3, 4, 6, 5, 6, 7, 5 ];
 
-};
+		this.setIndex( index );
+		this.setAttribute( 'position', new Float32BufferAttribute( positions, 3 ) );
+		this.setAttribute( 'uv', new Float32BufferAttribute( uvs, 2 ) );
 
-LineSegmentsGeometry.prototype = Object.assign( Object.create( InstancedBufferGeometry.prototype ), {
+		this.isLineSegmentsGeometry = true;
 
-	constructor: LineSegmentsGeometry,
+		this._box = new Box3();
+		this._vector = new Vector3();
```

Comment:
Internal value formerly in closure functions can be declared at the beginning of the file.
see:
https://github.com/mrdoob/three.js/commit/0aeeb7dae8ba66f2d17f115300b35a6c83fb119e#diff-ec68f22f36a17164468f6175039b8dd8